### PR TITLE
zh: replace the usage of DM-Ansible with TiUP

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- remove `deploy-a-dm-cluster-using-ansible.md` and `cluster-operations.md`
- replace other usage of DM-ansible with TiUP

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [x] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@TomShawn @GMHDBJD PTAL

File: zh/manually-upgrade-dm-1.0-to-2.0.md [hunk 1]
```diff
@@ -103,11 +103,11 @@ from:
 
 ## 第 2 步：部署 v2.0.x 集群
 
-使用 TiUP 按所需要节点数部署新的 v2.0.x 集群。
+[使用 TiUP](deploy-a-dm-cluster-using-tiup.md) 按所需要节点数部署新的 v2.0.x 集群。
 
 ## 第 3 步：下线 v1.0.x 集群
 
-如果原 v1.0.x 集群是使用 DM-Ansible 部署的，则[使用 DM-Ansible 下线 v1.0.x 集群](https://docs.pingcap.com/zh/tidb-data-migration/stable/cluster-operations#%E4%B8%8B%E7%BA%BF%E9%9B%86%E7%BE%A4)。
+如果原 v1.0.x 集群是使用 DM-Ansible 部署的，则[使用 DM-Ansible 下线 v1.0.x 集群](https://docs.pingcap.com/zh/tidb-data-migration/v1.0/cluster-operations#%E4%B8%8B%E7%BA%BF%E9%9B%86%E7%BE%A4)。
```
Comment (GMHDBJD) [inline zh/manually-upgrade-dm-1.0-to-2.0.md:110]:
```suggestion
如果原 v1.0.x 集群是使用 DM-Ansible 部署的，则[使用 DM-Ansible 下线 v1.0.x 集群](https://docs.pingcap.com/zh/tidb-data-migration/v1.0/cluster-operations#下线集群)。
```

**Review by GMHDBJD — COMMENTED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**